doc/Jamfile.v2

```diff
@@ -21,9 +21,7 @@ import generators ;
 import sequence ;
 import path ;
 import "class" : is-a ;
-
 path-constant BOOST_DOC : . ;
```

Comment:
@Anupam-tripathi You only removed two blank lines with an insignificant title of PR. This is **not** a good practice. I suggest you to read **contribution guidelines** of boost and then work on an issue.